docs/developer_guide.md

```diff
@@ -0,0 +1,145 @@
+
+All of our code are open sourced. Developers can follow this guide to get projects set up.
+
+### Prerequisite
+
+1. Download `qtum-0.14.13-osx64.tar.gz`  from [https://github.com/qtumproject/qtum/releases](https://github.com/qtumproject/qtum/releases) and unzip it.
+
+2. Then open terminal and enter the directory:
+
+      
+         $ cd /qtum-0.14.13/bin
+      
+
+3. Run command to start sync.
+
+       
+         $ sudo ./qtumd -testnet -rpcuser=bodhi -rpcpassword=bodhi -logevents -printtoconsole
```

Comment:
sudo shouldn't be necessary here.

Reply:
Without sudo, it would say permission denied. Maybe it depends on where your folder is put.

Reply:
I would suggest extracting the qtum files in your home (users) dir: ie. /Users/deric/ so then you dont have to sudo